### PR TITLE
BI-4784: add mutation functions to supported functions list

### PR DESCRIPTION
**Pull request by AdayevKP**

### CODE REVIEW

File: lib/dl_formula/dl_formula/mutation/registry.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import typing as tp
+
+from dl_api_lib.query.registry import is_forkable_dialect
```
Comment (KonstantAnxiety) [inline lib/dl_formula/dl_formula/mutation/registry.py:3]: dl_formula can't depend on dl_api_lib

File: lib/dl_formula/dl_formula/mutation/registry.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+from dl_formula.core.dialect import DialectCombo
+from dl_formula.mutation.lookup import LOOKUP_MUTATOR_REGISTRY
+
+
+def get_mutation_functions_names(dialect: DialectCombo) -> tp.List[str]:
```
Comment (KonstantAnxiety) [inline lib/dl_formula/dl_formula/mutation/registry.py:8]: We tend to use just `list[str]` in type hints

File: lib/dl_api_lib/dl_api_lib/connector_registrator.py [hunk 3]
```diff
@@ -59,6 +60,7 @@ def register_connector(cls, connector: Type[ApiConnector]) -> None:
                 factory_cls=mqm_setting_item.factory_cls,
             )
         register_is_forkable_source(backend_type=backend_type, is_forkable=connector.is_forkable)
```
Comment (KonstantAnxiety) [inline lib/dl_api_lib/dl_api_lib/connector_registrator.py:62]: @altvod, take a look please
This is unused at the moment, do we need this?

File: lib/dl_formula/dl_formula/mutation/registry.py (hunk 2 above)
Comment (AdayevKP) [reply lib/dl_formula/dl_formula/mutation/registry.py]: fixed

File: lib/dl_formula/dl_formula/mutation/registry.py (hunk 1 above)
Comment (AdayevKP) [reply lib/dl_formula/dl_formula/mutation/registry.py]: fixed

File: lib/dl_api_lib/dl_api_lib/connector_registrator.py (hunk 3 above)
Comment (altvod) [reply lib/dl_api_lib/dl_api_lib/connector_registrator.py:62]: If it's not used, then let's remove it

Comment (AdayevKP) [reply lib/dl_api_lib/dl_api_lib/connector_registrator.py]: done

**Review by KonstantAnxiety — APPROVED** [review]

**Pull request merged**